scripts/feedme.coffee

```diff
@@ -42,13 +42,9 @@ class Fabrik
     menuFromPast : "the menu is outdated"
     menuFromFuture : "the menu is from the future"
     resting : "fabrik is on a day off"
-    holiday : "fabrik is probably on holiday, fall back to manual check"
 
   constructor: (@robot) ->
 
-  checkHoliday: (text) =>
-    text.toLowerCase().indexOf(@holidayMagic) > 0
```

Comment:
You remove the one and only reference to `@holidayMagic` here, so I think it would be appropriate to remove `@holidayMagic` altogether to make this changeset more coherent.